### ISSUE TITLE
License artworks

### ISSUE THREAD

**Issue by TheEvilSkeleton** [Vanilla-OS/vanilla-backgrounds#7]
This repo does not contain a license file, which means that they are legally proprietary. Since these are creative work, perhaps we could look at [CC BY-SA 4.0](https://creativecommons.org/licenses/by-sa/4.0/)? Derivative work must be open and licensed under the same license.

**Comment by kbdharun** [comment]
I agree with this. It seems best to have a license file for the artworks.

**Comment by mirkobrombin** [comment]
They are licensed in the package https://github.com/Vanilla-OS/vanilla-backgrounds/blob/main/debian/copyright
but I agree with a LICENSE file
